lib/mp4/probe.js

```diff
@@ -153,24 +153,30 @@ startTime = function(timescale, fragment) {
       scale = timescale[id] || 90e3;
 
       // get the base media decode time from the tfdt
-      baseTime = findBox(traf, ['tfdt']).map(function(tfdt) {
-        var version, result;
-
-        version = tfdt[0];
-        result = toUnsigned(tfdt[4] << 24 |
-                            tfdt[5] << 16 |
-                            tfdt[6] <<  8 |
-                            tfdt[7]);
-        if (version ===  1) {
-          result *= Math.pow(2, 32);
-          result += toUnsigned(tfdt[8]  << 24 |
-                               tfdt[9]  << 16 |
-                               tfdt[10] <<  8 |
-                               tfdt[11]);
-        }
-        return result;
-      })[0];
-      baseTime = baseTime || Infinity;
```

Comment:
So, what you're saying is that baseTime can have a value of zero and we shouldn't set it to Infinity in this case which this line does.
I think we can simplify this change by just updating this line to something like
```js
if (typeof baseTime !== 'number') {
  baseTime = Infinity;
}
```
instead of any other change